tests/wicked/basic/ref/t04_dynamic_addresses_legacy.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Set up dynamic addresses from legacy ifcfg files
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+
+use base 'wickedbase';
+use strict;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    record_info('Info', 'Set up dynamic addresses from legacy ifcfg files');
+}
+
+sub test_flags {
+    return {always_rollback => 1};
```

Comment:
hm , maybe we can remove this flag from ref VMs ? totally or at least for such scenarios ?

Reply:
needed to sync all the tests